lib/bulkhead/Bulkhead.ts

```diff
@@ -0,0 +1,54 @@
+import { DEFAULT_SIZE } from './BulkheadOptions';
+
+export class Bulkhead {
+
+  private inFlight: number = 0;
```

Comment:
`inFlight`... to the skyyyyy... :-).
Just kidding, I think `inFlight` is not very appropriate here, maybe other options like:
- `inExecution`
- `activeRuns`
- `awaitingRuns`
etc.